eq-author/src/App/page/Routing/RoutingPage/RoutingEditor/RuleEditor/index.js

```diff
@@ -91,68 +104,69 @@ export class UnwrappedRuleEditor extends React.Component {
     const existingRadioConditions = {};
 
     return (
-      <div data-test="routing-rule">
-        <Box>
-          {title && <Title>{title}</Title>}
-          <Buttons>
-            <Button
-              onClick={this.handleDeleteClick}
-              data-test="btn-delete"
-              variant="tertiary"
-              small
-            >
-              <IconText icon={IconRoute}>Remove rule</IconText>
-            </Button>
-          </Buttons>
-          <div>
-            <TransitionGroup>
-              {expressions.map((expression, index) => {
-                const component = (
-                  <Transition key={expression.id} exit={false}>
-                    <BinaryExpressionEditor
-                      expression={expression}
-                      label={index > 0 ? "AND" : "IF"}
-                      isOnlyExpression={expressions.length === 1}
-                      canAddAndCondition={
-                        !existingRadioConditions[get("left.id", expression)]
-                      }
-                    />
-                  </Transition>
-                );
-                if (get("left.type", expression) === RADIO) {
-                  existingRadioConditions[get("left.id", expression)] = true;
-                }
-                return component;
-              })}
-            </TransitionGroup>
-            <Grid align="center">
-              <CenteringColumn gutters={false} cols={1}>
-                <TextButton
-                  onClick={this.handleCreateExpressionClick}
-                  data-test="btn-and"
-                >
-                  AND
-                </TextButton>
-              </CenteringColumn>
-            </Grid>
-          </div>
-          <DestinationSelector
-            id={rule.id}
-            label="THEN"
-            onChange={this.handleDestinationChange}
-            value={destination}
-            data-test="select-then"
-          />
-        </Box>
-      </div>
+      <Rule data-test="routing-rule" className={className}>
+        <Header>
+          <Label inline>
+            Match
+            <SmallSelect name="match" id="match">
+              <option value="all" selected>
+                All of
+              </option>
+              <option value="any" disabled>
```

Comment:
Is it worth making it more obvious its disabled until its functional? To me, it looked as though All of was in bold because its selected and Any of was a selectable option. Same for Answer/Metadata option.

Reply:
Fair point 👍